devito/profiler.py

```diff
@@ -81,18 +86,23 @@ def add_profiling(self, code, name, byte_size=4, omp_flag=None):
                           Defaults to 4.
         :param omp_flag: OpenMP flag to add before profiling operations,
                          if needed.
+        :param to_ignore: List of strings containing the labels of
+                          symbols used as loop variables
```

Comment:
Hmm, in that case, yes that would be possible. I'm still not sure though what we would gain from a global approach over querying the relevant `Iteration` and `Expression` hierarchy explicitly in the `Operator `and passing the information to it's profiler - once the internals always use them that is, of course.